src/Generation/IconGenerator.php

```diff
@@ -40,7 +40,7 @@ public function generate(): void
                 $this->filesystem->copy($file->getRealPath(), $pathname);
 
                 if (is_callable($set['after'] ?? null)) {
-                    $set['after']($pathname, $set);
+                    $set['after']($pathname, $set, $file);
```

Comment:
@swapnilsarwe my only thought is around if you want just a string of the path, or the Spl object.
Not sure if it's consequential either way, but could be something worth considering.

Reply:
Sure. It works for me - atleast for my current use case. I can simply pass the string `$file->getRealPath()` instead of object of type `SplFileObject`.

I was just being little greedy by passing the complete `$file` object.

I will make the changes and update the PR.

Reply:
Have made the changes in the PR